### PR TITLE
Fix photo upload handling in Django.

### PR DESCRIPTION
**Pull request by nworden**
Django handles uploaded files differently than webapp2.
Also refactors create.py to import the photo module. This is what's
recommended by the style guide, and this is a convenient (by convenient,
I mean mandatory) time to do that because the mocking library doesn't
seem to handle the situation well otherwise.

### CODE REVIEW

File: app/utils.py [hunk 1]
```diff
@@ -296,6 +296,17 @@ def validate_image(bytestring):
         return False
 
 
+def validate_django_image(uploaded_file):
+    try:
+        image = None
+        if uploaded_file:
+            image = images.Image(uploaded_file.read())
+            image.width
```
Comment (eliahhecht) [inline app/utils.py]: this line looks weird. are you just making this call to validate that the image was constructed properly? if it wasn't, would we even reach this line?

File: app/utils.py [hunk 2]
```diff
@@ -296,6 +296,17 @@ def validate_image(bytestring):
         return False
 
 
+def validate_django_image(uploaded_file):
```
Comment (eliahhecht) [inline app/utils.py]: add documentation for what this method returns?

File: app/utils.py (hunk 1 above)
Comment (nworden) [reply app/utils.py]: That's my assumption, yeah. This function is really just a Django variant on an existing function (validate_image, above); I added a note about that in the comment.

File: app/utils.py (hunk 2 above)
Comment (nworden) [reply app/utils.py]: Done.

**Pull request merged**